### PR TITLE
Credential Rotation Fixes

### LINKED ISSUE #1580: [master] Credential rotation option `SSH Key Pair for Worker Nodes` is shown for workerless Shoots
**What happened**:
The credential rotation option `SSH Key Pair for Worker Nodes` is shown for workerless Shoots, however this does not make sense as there are no worker nodes.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know**:

**Environment**:

### LINKED ISSUE #1581: [master] Improve Credential Rotation layout
**What would you like to be added**:
Layout of version `1.69.2`
Screenshot 2023-09-25 at 17 03 57

Layout of current `master` branch
Screenshot 2023-09-25 at 17 06 42

They layout has to be improved so that the title and description are visually grouped together. Currently it seems as if the description of the previous row is grouped together with the title of the next row.

**Why is this needed**:

### PR DESCRIPTION
**Pull request by grolu**
- Do not show node ssh key pair rotation for workerless clusters
- Hide disabled actions messages
- Fixed tooltips for disabled g-shoot-action-buttons
- Fixed layout by removing padding from g-list-item-content description slot

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #1580
Fixes #1581

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/components/GCredentialTile.vue [hunk 1]
```diff
@@ -14,7 +14,9 @@ SPDX-License-Identifier: Apache-2.0
         mdi-key-change
       </v-icon>
     </template>
-    <g-list-item-content>
+    <g-list-item-content
+      description-class="no-padding"
```
Comment (petersutter) [inline frontend/src/components/GCredentialTile.vue:18]:
```suggestion
      description-class="no-padding text-medium-emphasis"
```

File: frontend/src/components/GListItemContent.vue [hunk 2]
```diff
@@ -19,6 +19,7 @@ SPDX-License-Identifier: Apache-2.0
   <div
     v-if="slots.description || props.description"
     class="text-body-2 text-high-emphasis pt-1"
+    :class="descriptionClass"
```
Comment (petersutter) [inline frontend/src/components/GListItemContent.vue:22]: alternatively have a dense prop to control the padding
also, why has the description text-high-emphasis and not text-medium-emphasis? cc @holgerkoser

Comment (holgerkoser) [reply frontend/src/components/GListItemContent.vue:22]: bug

File: frontend/src/components/GListItemContent.vue [hunk 3]
```diff
@@ -38,5 +39,8 @@ const props = defineProps({
   description: {
     type: String,
   },
+  descriptionClass: {
```
Comment (holgerkoser) [inline frontend/src/components/GListItemContent.vue:42]: not needed

File: frontend/src/components/GShootActionRotateCredentials.vue [hunk 4]
```diff
@@ -384,10 +384,10 @@ export default {
         successMessage: `Preparing credential rotation for ${this.shootName}`,
         heading: 'Do you want to prepare the rotation of all credentials?',
         actions: [
-          ...componentTexts['rotate-kubeconfig-credentials'].actions,
+          ...this.shootEnableStaticTokenKubeconfig ? componentTexts['rotate-kubeconfig-credentials'].actions : [],
```
Comment (holgerkoser) [inline frontend/src/components/GShootActionRotateCredentials.vue:387]:
```suggestion
          ...(this.shootEnableStaticTokenKubeconfig 
            ? componentTexts['rotate-kubeconfig-credentials'].actions 
            : []),
```

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**